app/views/sessions/new.html.erb

```diff
@@ -6,7 +6,8 @@
          <%= text_field_tag :email %><br><br>
          <%= label_tag :password%>
          <%= password_field_tag :password%><br><br>
-         <%= submit_tag "Log In"%>
+         <%= submit_tag "Log In", class: 'welcome-button'%>
```

Comment:
Can these buttons be the same as the ones on the timesheet page please (the smaller grey ones)

Reply:
Done